test/phoenix/router/forward_test.exs

```diff
@@ -0,0 +1,144 @@
+defmodule Phoenix.Router.ForwardTest do
+  use ExUnit.Case, async: true
+  use RouterHelper
+
+  defmodule Controller do
```

Comment:
What I would do is to make the Controller as pass through (it doesn't do anything) and call the helpers inside the test themselves. It is going to make the test clearer because I don't need to look at the controller then at the test to know what is being generated and what I am asserting on.